raiden_contracts/utils/type_aliases.py

```diff
@@ -1,4 +1,26 @@
 from typing import NewType
```

Comment:
Should we import `web3.Nonce` here as well, so we can use it from that module as well?

Reply:
I thought it would be better if we imported that directly from web3, so that it is easier to spot which types are actually the same. But I have not strong opinion either way.

Reply:
No strong feelings either, let's keep it like this then.